protocol-units/execution/opt-executor/src/executor/initialization.rs

```diff
@@ -167,7 +167,7 @@ impl Executor {
 		// replace the db path with a temporary directory
 		let value = tempfile::tempdir()?.into_path(); // todo: this works because it's at the top level, but won't be cleaned up automatically
 		maptos_config.chain.maptos_db_path.replace(value);
+		dbg!(&maptos_config.chain.maptos_db_path);
```

Comment:
Remember to remove this, unless you have a good reason to keep it.

Reply:
Done in the subsequent commit.